docs/components/camera/webcam.md

```diff
@@ -133,16 +133,25 @@ Viam supports the following pixel formats:
 
 - I420
 - I444
-- MJPEG
+- MJPEG / MJPG
 - NV12
 - NV21
 - RGBA
-- UYVY
-- YUY2
+- UYVY / Y422
+- YUY2 / YUYV / V422
 - Z16
 
 If your machine is connected to the Viam app, the available pixel formats supported by your camera automatically appear in the **Format** dropdown menu, which is visible when you click the **Show more** button.
 
+On Linux, you can also manually determine which pixel formats your camera supports by running the following command:
```

Comment:
```suggestion
On Linux, you can also manually determine which pixel formats your camera supports by running the following command on the machine your camera is connected to:
```
And then remove line 153?

Reply:
(haha I had exactly this copy in place before switching it around. You're right though, this is better!!)